README.md

```diff
@@ -78,6 +78,15 @@ Follow the given instructions for Login into the app.
 
 To run the tests run: `python manage.py test`.
 
+## Documentation
+
+- Join us on [Open session](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs/topic/Open.20Sessions) and see [Meeting minutes](https://docs.google.com/document/d/1YF13IbBrU1ln4ZF1fOpgb-xGRgIF6tZLSjIBQgDmN7k/edit).
```

Comment:
```suggestion
- Read the [Meeting minutes](https://docs.google.com/document/d/1YF13IbBrU1ln4ZF1fOpgb-xGRgIF6tZLSjIBQgDmN7k/edit) notes from our open sessions about the project.
```

Perhaps the Open Session mention could go to Contributing section. Also, topics can be easily changed, so it's easy for this link to become invalid. But we can still link it on another session, documentation isn't available on this topic.

Reply:
Thanks! I will do the necessary changes.